cmd/labs/project/fetcher.go

```diff
@@ -140,9 +145,21 @@ func (f *fetcher) checkReleasedVersions(cmd *cobra.Command, version string) (str
 	return version, nil
 }
 
-func (i *fetcher) loadRemoteProjectDefinition(cmd *cobra.Command, version string) (*Project, error) {
+func (i *fetcher) loadRemoteProjectDefinition(cmd *cobra.Command, version string, offlineInstall bool) (*Project, error) {
 	ctx := cmd.Context()
-	raw, err := github.ReadFileFromRef(ctx, "databrickslabs", i.name, version, "labs.yml")
+	var raw []byte
+	var err error
+	if !offlineInstall {
+		raw, err = github.ReadFileFromRef(ctx, "databrickslabs", i.name, version, "labs.yml")
+	} else {
+		libDir, file_err := PathInLabs(ctx, i.name, "lib")
+		if file_err != nil {
+			return nil, file_err
+		}
+		fileName := filepath.Join(libDir, "labs.yml")
+		raw, err = os.ReadFile(fileName)
+	}
+
 	if err != nil {
 		return nil, fmt.Errorf("read labs.yml from GitHub: %w", err)
 	}
```

Comment:
The error handling here doesn't match the error above.

You can fully separate the paths for offline and online to make this more clear.